solution/ui/regulations/utilities/api.js

```diff
@@ -196,12 +196,13 @@ function fetchJson({
             if (_isBoolean(isOk) && !isOk) {
                 throw parseError({ ...json, status: httpStatus });
             } else {
-                json.expiration_date = Date.now() + 8 * 60 * 60 * 1000; // 24 hours * 60 minutes * 60 seconds * 1000
-                if (cacheResponse)
+                if (cacheResponse) {
+                    json.expiration_date = Date.now() + 8 * 60 * 60 * 1000; // 24 hours * 60 minutes * 60 seconds * 1000
```

Comment:
if we're not caching an API response, don't add the `experation_date` field.  It's not necessary